<!--launch file-->

<launch>
	<param name="use_sim_time" value="true" />
	<!--include file="$(find launch)/config/server.launch" /-->
	
	 <!--arg name="robot_name" value="$(optenv NIMBRO_ROBOT_NAME)" /-->
	 <arg name="robot_name" value="xs4_yf_bonnfield1_2" />
	 

        <node name="config_server" pkg="config_server" type="config_server" output="screen" required="true">
                <param name="config_path" value="$(find visual_tracking)/config" />
                <param name="robot_name" value="$(arg robot_name)" />
        </node>
	
        
	
       <!-- start the vision localization node -->
	<node  pkg="visual_tracking" type="visual_tracking_node" name="visual_tracking" output="screen"> <!--launch-prefix="xterm -e gdb **args"-->
	</node>
	
	        
        <!-- start bagfile -->
        <node name="rosbag" pkg="rosbag" type="play" args="--clock -r 1.0 $(find visual_tracking)/../bags/normal_test/1.2.bag"> 
         </node>
	
</launch>
